user_docs/en/changes.md

```diff
@@ -2,21 +2,41 @@
 
 ## 2024.4
 
-### Important notes
+This release includes a number of improvements to braille.
```

Comment:
To me, putting this first, and listing off all of the braille changes, makes it seem like that was the focus of this release, and was all that changed.
Since there are so many braille changes, and miscellaneous other changes, I would suggest putting the list of braille stuff last.

Reply:
Done, let me know what you think.